docs/source/visualisation/experiment_tracking.md

```diff
@@ -1,49 +1,66 @@
 
-- **Kedro** - If you need experiment tracking, are looking for improved metrics visualisation and want a lightweight tool created to leverage existing functionality in Kedro. Kedro does not support a model registry.
+- **Kedro** - If you need experiment tracking, are looking for improved metrics visualisation and want a lightweight tool to work alongside existing functionality in Kedro. Kedro does not support a model registry.
 - **MLflow** - You can combine MLFlow with Kedro by using [`kedro-mlflow`](https://kedro-mlflow.readthedocs.io/en/stable/) if you require experiment tracking, model registry and/or model serving capabilities or have access to Managed MLflow within the Databricks ecosystem.
-- **Neptune** - If you require experiment tracking and model registry functionality, improved visualisation of metrics and support for collaborative data science. Leverage [`kedro-neptune`](https://docs.neptune.ai/integrations/kedro/) for your workflow.
+- **Neptune** - If you require experiment tracking and model registry functionality, improved visualisation of metrics and support for collaborative data science, you may consider [`kedro-neptune`](https://docs.neptune.ai/integrations/kedro/) for your workflow.
 
-We support [a growing list of integrations](../extend_kedro/plugins.md).
+[We support a growing list of integrations](../extend_kedro/plugins.md).
 
 ## Set up a project
 
-To enable the experiment tracking features of Kedro-Viz you need to:
+This section describes the steps necessary to set up experiment tracking and access logged metrics, using the [spaceflights tutorial](../tutorial/spaceflights_tutorial.md) with a version of Kedro equal to or higher than 0.18.4, and a version of Kedro-Viz equal to or higher than 5.2.
+
+
+There are three steps to enable experiment tracking features with Kedro-Viz. We illustrate how to:
 
-- [Set up a session store to capture experiment metadata](#set-up-the-session-store),
-- [Set up experiment tracking datasets to list the metrics to track](#set-up-tracking-datasets),
-- [Modify your nodes and pipelines to output those metrics](#set-up-your-nodes-and-pipelines-to-log-metrics).
+- [Set up a session store to capture experiment metadata](#set-up-the-session-store)
+- [Set up experiment tracking datasets to list the metrics to track](#set-up-experiment-tracking-datasets)
+- [Modify your nodes and pipelines to output those metrics](#modify-your-nodes-and-pipelines-to-log-metrics)
 
-This page describes the steps necessary to set up experiment tracking and access logged metrics, using the [spaceflights tutorial](../tutorial/spaceflights_tutorial.md).
+### Prerequisites
+We assume that you have already [installed Kedro](../get_started/install.md) and [Kedro-Viz](../visualisation/kedro-viz_visualisation.md). Before proceeding, ensure you're using Kedro-Viz `>=5.2.1` (you can confirm your Kedro-Viz version by running `kedro info`).
```

Comment:
This is tricky. Experiment tracking worked back to version `4.1.1`, as you removed from somewhere else on the page.

If we want to state when plots came to the product, that was with the release of `5.0.0`.

I'm not sure what's correct to put here, though `5.2.1` doesn't feel right.

Reply:
This is in the `Prerequisites` section though, so isn't this is the highest-level requirement for getting experiment tracking in Viz, not just the most up to date?

Reply:
It's the pre-requisites for the code tutorial, which is definitely 5.2.1 but I can make that clearer. And I can add a "what is in what version?" section to clarify.

Reply:
@tynandebold Please check this latest commit https://github.com/kedro-org/kedro/pull/2292/commits/decfc71cd6dfd3fce79aa41bc6d1400e1d1f05a4 for the revisions I've put in to cover the versions and what are supported. Please can you confirm they are correct or make the necessary changes?

Reply:
You can see the new copy here https://stichbury.github.io/visualisation/experiment_tracking.html